.travis.yml

```diff
@@ -7,7 +7,10 @@ before_install:
   - export CHROME_BIN=chromium-browser
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
+#  - pip install --user codecov
```

Comment:
Could you please leave a comment above these commented out lines that points back to this PR for context?

Reply:
Good idea, done!